docs/guides/protect-page-login/php.mdx

```diff
@@ -0,0 +1,135 @@
+
+Therefore, Ory manages Anti-CSRF Cookies as well as Ory Session Cookies for you.
+That however requires that Ory and your application run on the same domain!
+
+If your application runs on `http://localhost:3000` then Ory needs to be
+available on the hostname `localhost` as well (e.g. `http://localhost:3001`).
+That is why we need the Ory CLI, because it has a proxy included which mirrors
+Ory's API endpoints on the domain of your application.
+
+[![Ory Proxy mirrors Ory's APIs](https://mermaid.ink/img/pako:eNptkMFKxEAMhl8l5GIXdm1BT3MQWlxEEBxWL8JchjbaYqcZMlO0bPfdna71sGBOP-H7EpIj1twQKvwQ61t4OpgBUlVZJfwVSDaw293ND_tXaGP0Ks97rm3fcojqtiiKGXT2LBNo4e9p8-vqs-I6EZYANkJ-zTLlM5T6cYGvwpLCBf3vgpvzgjJ741Gg9H41ykU4UPA8BJr1OuaiWeEWHYmzXZNOOy6IwdiSI4MqxcbKp0EznBI3-sZG2jddZEH1bvtAW7Rj5JdpqFFFGekPuu9sepNbqdMP2FdlPQ)](https://mermaid-js.github.io/mermaid-live-editor/edit#pako:eNptkMFKxEAMhl8l5GIXdm1BT3MQWlxEEBxWL8JchjbaYqcZMlO0bPfdna71sGBOP-H7EpIj1twQKvwQ61t4OpgBUlVZJfwVSDaw293ND_tXaGP0Ks97rm3fcojqtiiKGXT2LBNo4e9p8-vqs-I6EZYANkJ-zTLlM5T6cYGvwpLCBf3vgpvzgjJ741Gg9H41ykU4UPA8BJr1OuaiWeEWHYmzXZNOOy6IwdiSI4MqxcbKp0EznBI3-sZG2jddZEH1bvtAW7Rj5JdpqFFFGekPuu9sepNbqdMP2FdlPQ)
+
+## Create an Entry Page
+
+This is a working example of basic `index.php` script which creates an Ory client,
+registers new route for our Dashboard and makes use of [Before Route Middlewares](https://github.com/bramus/router#before-router-middlewares)
+to validate if the user is allowed to view the Dashboard.
+
+```mdx-code-block title="index.php"
+import indexPHP from '!!raw-loader!../../../code-examples/protect-page-login/php/index.php'
+import CodeBlock from '@theme/CodeBlock'
+
+<CodeBlock language="php">{indexPHP}</CodeBlock>
+```
+
+## Validate and Login
+
+Next we will create handler which will check with your Ory project if the
+user has a valid session. Notice here that we are taking the current `request`
+cookies and passing them along to the Ory client.
+
+If the session is not valid the request is redirected to the Ory project for
+login. At this point we have not set up any custom UI management and thus will
+be shown the Ory Managed UI login page.
+
+For the last part we need to add the Dashboard handler (the page we would like to
+protect) which will render an HTML with the session data.
+
+This is accomplished by the simple `App` class stored in the `app.php` file:
+```mdx-code-block title="app.php"
+import appPHP from '!!raw-loader!../../../code-examples/protect-page-login/php/app.php'
+
+<CodeBlock language="php">{appPHP}</CodeBlock>
+```
+
+## Run your App
+
+Start your HTTP server and access the proxy URL
+
+```shell-session
+php -S 127.0.0.1:3000
+```
+
+```mdx-code-block
+import SdkEnvVar from '@site/src/components/SdkEnvVar'
+
+<SdkEnvVar />
+```
+
+```shell-session
+ory proxy http://localhost:3000
+```
+
+Then open [http://localhost:4000](http://localhost:4000) in your browser. You
+are presented with Ory's Sign In page! Let's click on sign up and create your
+first user!
```

Comment:
I find it a bit confusing to start the server on `:3000` previously, but then going to `localhost:4000`. Can you maybe add something the lines of

> ... access the php app through the proxy ...

Reply:
right, I changed it to:
```To access the PHP app through the ORY proxy open```